VxWorks build fails

The VxWorks build fails with the following error:

```
In file included from ../PIGCSController.cpp:24:
../PIE727Controller.h:47: error: ISO C++ forbids initialization of member 'm_CCL_ADVANCED_ID'
../PIE727Controller.h:47: error: making 'm_CCL_ADVANCED_ID' static
../PIE727Controller.h:48: error: ISO C++ forbids initialization of member 'm_CCL_DEFAULT_ID'
../PIE727Controller.h:48: error: making 'm_CCL_DEFAULT_ID' static
../PIE727Controller.h:51: error: ISO C++ forbids initialization of member 'm_CCL_ADVANCED_PWD'
../PIE727Controller.h:51: error: making 'm_CCL_ADVANCED_PWD' static
../PIE727Controller.h:51: error: invalid in-class initialization of static data member of non-integral type 'const char*'
make[3]: *** [PIGCSController.o] Error 1
make[3]: Leaving directory `/path/to/epics/synApps_6_2_1/support/motor-git/modules/motorPIGCS2/pigcs2App/src/O.vxWorks-ppc32'
make[2]: *** [install.vxWorks-ppc32] Error 2
make[2]: Leaving directory `/path/to/epics/synApps_6_2_1/support/motor-git/modules/motorPIGCS2/pigcs2App/src'
make[1]: *** [src.install] Error 2
make[1]: Leaving directory `/path/to/epics/synApps_6_2_1/support/motor-git/modules/motorPIGCS2/pigcs2App'
make: *** [pigcs2App.install] Error 2
```

--- a/pigcs2App/src/PIE727Controller.cpp
+++ b/pigcs2App/src/PIE727Controller.cpp
@@ -23,6 +23,10 @@ Created: 15.12.2010
 PIE727Controller::PIE727Controller(PIInterface* pInterface, const char* szIDN)
 : PIGCSPiezoController (pInterface, szIDN)
 {
+    this->m_CCL_ADVANCED_ID = 1;   // advanced user ID
+    this->m_CCL_DEFAULT_ID  = 0;   // default user ID
+    strcpy(this->m_CCL_ADVANCED_PWD, "advanced");   // advanced user password
+    
     this->m_hasqFRF=false; ///< is "FRF?" command available
     this->m_hasqTRS=false; ///< is "TRS?" command available
     this->m_hasqSRG=false;

--- a/pigcs2App/src/PIE727Controller.h
+++ b/pigcs2App/src/PIE727Controller.h
@@ -47,7 +47,7 @@ class PIE727Controller : public PIGCSPiezoController
 		unsigned int m_CCL_ADVANCED_ID;   // advanced user ID
 		unsigned int m_CCL_DEFAULT_ID;   // default user ID
 
-		char m_CCL_ADVANCED_PWD[8];   // advanced user password
+		char m_CCL_ADVANCED_PWD[9];   // advanced user password
 };
 
 #endif /* PIE727CONTROLLER_H_ */

--- a/pigcs2App/src/PIE727Controller.h
+++ b/pigcs2App/src/PIE727Controller.h
@@ -44,11 +44,10 @@ class PIE727Controller : public PIGCSPiezoController
 
 		asynStatus setGCSCmdLvl(unsigned int cmdLvl);
 		
-		const unsigned int m_CCL_ADVANCED_ID = 1;   // advanced user ID
-		const unsigned int m_CCL_DEFAULT_ID  = 0;   // default user ID
+		unsigned int m_CCL_ADVANCED_ID;   // advanced user ID
+		unsigned int m_CCL_DEFAULT_ID;   // default user ID
 
-
-		const char *m_CCL_ADVANCED_PWD  = "advanced";   // default user ID
+		char m_CCL_ADVANCED_PWD[8];   // advanced user password
 };
 
 #endif /* PIE727CONTROLLER_H_ */

